README.md

```diff
@@ -1,4 +1,6 @@
-# Terratag by env0 [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
+[![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
+
+# [<img src="ttlogo.png" width="300">](https://terratag.io) 
```

Comment:
Sorry to be a stickler but can we have the logo above the badges ? that's usually the format.
Also can we add an `alt` to the `img` ?

Reply:
No worries, I’ll take care of this.

Reply:
Moved the logo, and added alt text.